integration-tests/src/test/resources/features/bridge.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Bridge tests
+
+  Scenario: Bridge is created and correctly deleted
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the list of Bridge instances is containing the Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+    
+    When delete the Bridge "mybridge"
+
+    Then the Bridge "mybridge" is not existing within 2 minutes
+    And the Ingress of Bridge "mybridge" is not available within 1 minute
+
+    And the Manager metric 'manager_bridge_status_change_total{status="PROVISIONING",}' count is at least 1
+    And the Manager metric 'manager_bridge_status_change_total{status="READY",}' count is at least 1
+    And the Manager metric 'manager_bridge_status_change_total{status="DELETED",}' count is at least 1
+
+  Scenario: Cannot access the list of Bridges without authentication
+    Given the list of Bridge instances is failing with HTTP response code 401
+
+  Scenario: Cannot create a Bridge without authentication
+    Given create a new Bridge "mybridge" is failing with HTTP response code 401
+
+  Scenario: Cannot access Bridge details without authentication
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+    When logout of Manager
+
+    Then get Bridge "mybridge" is failing with HTTP response code 401
```

Comment:
Can you please add a test deleting non existing bridge?

Reply:
added. See https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/394/commits/895705998957de1b5329aff1430855cdfa09648e